README.md

```diff
@@ -0,0 +1,53 @@
+# Grafana Dashboard for Curity Identity Server
+
+Curity Identity Server publishes prometheus-compliant metrics. Grafana is a tool that can be used to visualize the metrics collected by Prometheus.
+This repository provides a preconfigured dashboard that shows the most important metrics of Curity Identity Server. Feel free to download the .json file and import it into your Grafana instance to view all the stats of your Curity Identity Server instance.
```

Comment:
Maybe change "of your Curity Identity Server instance" to "of your Curity Identity Server cluster" ?